spec/models/need_spec.rb

```diff
@@ -18,4 +18,61 @@
     it {should validate_presence_of(:supporters_needed)}
     it {should validate_presence_of(:status)}
   end
+
+
+  describe 'methods' do
+    it 'upcoming_active' do
```

Comment:
In the future, it may be helpful to include more details `it` statements here to describe what we're testing (e.g. '.upcoming_active returns all needs where end date has not yet passed`. A tip Brian gave me is to aim for testing to read like documentation!